src/schema.json

```diff
@@ -12,9 +12,10 @@
     "img-src": ["img.jpg", "img2.jpg"],
```

Comment:
This file will need to change to accommodate the incoming changes from Authentication branch